src/screens/DetailsScreen/DetailsScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import {RouteProp} from '@react-navigation/native';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {
+  SafeAreaView,
+  StatusBar,
+  Text,
+  useColorScheme,
+  View,
+} from 'react-native';
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+import {StackParamList} from 'types/navigation';
+import {CityItem} from '../../components';
+import styles from './DetailsScreenStyles';
+
+type DetailsProps = NativeStackScreenProps<StackParamList, 'Details'>;
+
+const DetailsScreen: React.FC<DetailsProps> = ({route}) => {
+  const {humidity, wspeed, pressure, cloud} = route.params;
+  const isDarkMode = useColorScheme() === 'dark';
+
+  const backgroundStyle = {
+    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
+  };
+
+  return (
+    <SafeAreaView style={backgroundStyle}>
+      <StatusBar
```

Comment:
yeah same here - status bar (if it does not contain different logic between screen) can be set once in App.tsx file